cfme/tests/infrastructure/test_providers.py

```diff
@@ -520,14 +521,16 @@ def test_compare_provider_templates(appliance, setup_provider_min_templates, pro
     Bugzilla:
         1746449
     """
-    templateList = []
-    my_slice = slice(0, min_templates, None)
-    view = navigate_to(provider, 'ProviderTemplates')
-    for t in view.entities.get_all(slice=my_slice):
-        t.ensure_checked()
-        templateList.append(t.name)
-    view.toolbar.configuration.item_select('Compare Selected Templates', handle_alert=True)
-    compare_templates_view = provider.create_view(TemplatesCompareView)
-    assert compare_templates_view.is_displayed
-    assert compare_templates_view.verify_checked_items_compared(templateList,
-                                                                compare_templates_view)
```

Comment:
This seems like a crucial part of the test case, why has it been removed?

Reply:
Thanks. Good catch. I left this off by mistake. Adding back.